Description: fix for node-schema-utils ≥ 3
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-11-18

--- a/src/index.js
+++ b/src/index.js
@@ -7,7 +7,7 @@
 import { SourceMapSource, RawSource, ConcatSource } from 'webpack-sources';
 import RequestShortener from 'webpack/lib/RequestShortener';
 import ModuleFilenameHelpers from 'webpack/lib/ModuleFilenameHelpers';
-import validateOptions from 'schema-utils';
+import { validate as validateOptions } from 'schema-utils';
 import schema from './options.json';
 import Runner from './uglify/Runner';
 
